components/brave_wallet_ui/page/screens/fund-wallet/fund-wallet.tsx

```diff
@@ -395,7 +401,7 @@ export const FundWalletScreen = () => {
                 onSubmit={nextStep}
                 disabled={!isNextStepEnabled}
                 isV2={true}
-                minWidth='436px'
+                minWidth='360px'
```

Comment:
Info: Updated for Android and [the size in design spec is (324px)](https://www.figma.com/file/muVcmcu4RFNu0LkXe9VRDl/Wallet-2.0?type=design&node-id=3390-259364&mode=dev).